services/api/config/application.yml.example

```diff
@@ -18,25 +18,35 @@ production:
   secret_token: ~
   blob_signing_key: ~
   sso_app_secret: ~
-  sso_app_id: ~
-  sso_provider_url: ~
-  workbench_address: ~
-  websocket_address: ~
+  sso_app_id: arvados-server
+  sso_provider_url: https://uuid_prefix.your.domain
```

Comment:
Should be `auth.your.domain` to match the install docs (except the parts of the install docs that say `sso.your-site.com`, which should also be fixed, whoops)